### ISSUE TITLE
question about 'bootstrapped_cross_entropy'

### ISSUE THREAD

**Issue by khgapa** [meetshah1995/pytorch-semseg#150]
I am using frrn. I have recently confirmed that the entire code has been modified and try to use it.
But, I got the error in 'bootstrapped_cross_entropy'.
Previously, the K value was in the frrn code.
Like this,
#################################################
def __init__(self, n_classes=21, model_type=None):
        super(frrn, self).__init__()
        self.n_classes = n_classes
        self.model_type = model_type
        self.K = 64 * 512
        self.loss = functools.partial(bootstrapped_cross_entropy2d, K=self.K)

        self.conv1 = utils.conv2DBatchNormRelu(3, 48, 5, 1, 2)
...
#################################################
In new code, some lines deleted and I can`t find how to give a 'K' value
Could you explain how to give a 'K' value
